doc/BUILD.rst

```diff
@@ -53,6 +53,7 @@ Required
 -  libzstd >= 1.4.0
 -  freetype2
 -  opusfile
+-  OpenSSL/LibreSSL (for a better SHA-256 implementation)
```

Comment:
This is an optional dependency. Other than that, everything looks fine! Sorry for taking so long to review this.